From 3ff1220251beff163e36f7d07c67b7a31f9848df Mon Sep 17 00:00:00 2001
From: Sean Cross <xobs@kosagi.com>
Date: Fri, 8 Jan 2016 11:53:16 +0800
Subject: [PATCH 28/65] spi: spidev: add novena to compatible list

The Novena FPGA is connected via SPI, and being an FPGA, it can
be reconfigured to be anything.  Therefore, it is easiest to
treat it as a generic SPI device, accessible from userspace.

Add kosagi,novena-fpga-spidev to the "compatible" list in order
to allow Novena to boot with a spidev device without generating
a warning on boot.

Signed-off-by: Sean Cross <xobs@kosagi.com>
---
 drivers/spi/spidev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/spi/spidev.c b/drivers/spi/spidev.c
index d0e7dfc..7d42069 100644
--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -695,6 +695,7 @@ static struct class *spidev_class;
 static const struct of_device_id spidev_dt_ids[] = {
 	{ .compatible = "rohm,dh2228fv" },
 	{ .compatible = "lineartechnology,ltc2488" },
+	{ .compatible = "kosagi,novena-fpga-spidev" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
-- 
2.7.3

